optional-clustering-analysis/clustering-calculations.R

```diff
@@ -138,9 +138,14 @@ if (!file.exists(opt$sce)){
   stop(paste(opt$sce, "does not exist."))
 }
 
-# Check that clustering type is valid 
-if (!opt$cluster_type %in% c("louvain", "walktrap")) {
-  stop("--cluster_type (-c) must be either louvain or walktrap.")
+# Split up string of cluster types
+cluster_types <- unlist(stringr::str_split(opt$cluster_types, ","))
```

Comment:
If you want to protect against someone adding a space, e.g.`louvain, walktrap`, we can do:


```suggestion
cluster_types <-  stringr::str_split(opt$cluster_types, ",") %>% unlist() %>% stringr::str_trim()
```